src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import core.basesyntax.constructors.Lottery;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall());
```

Comment:
A new Lottery instance is still created for each loop step. It's better to create one Lottery instance before the loop entrance.

Reply:
Bro <3